demo-vue/config/prod.env.js

```diff
@@ -0,0 +1,4 @@
+'use strict'
+module.exports = {
+  NODE_ENV: '"production"'
```

Comment:
Are the 2 sets of quotes needed here?

Reply:
Very good question. This is generated by vue-cli so I'm not 100% sure.